otter/test/convergence/test_gathering.py

```diff
@@ -320,31 +321,54 @@ class ExtractDrainedTests(SynchronousTestCase):
     """
     Tests for :func:`otter.convergence.extract_CLB_drained_at`
     """
-    summary = ("Node successfully updated with address: "
-               "'10.23.45.6', port: '8080', weight: '1', "
-               "condition: 'DRAINING'")
-    updated = '2014-10-23T18:10:48.001Z'
-    feed = (
-        '<feed xmlns="http://www.w3.org/2005/Atom">' +
-        '<entry><summary>{}</summary><updated>{}</updated></entry>' +
-        '<entry><summary>else</summary><updated>badtime</updated></entry>' +
-        '</feed>')
+    updated_summary = ("Node successfully updated with address: "
+                       "'10.23.45.6', port: '8080', weight: '1', "
+                       "condition: 'DRAINING'")
+    created_summary = ("Node successfully created with address: "
+                       "'10.23.45.6', port: '8080', condition: 'DRAINING', "
+                       "weight: '1'")
+    updated1 = '2014-10-23T18:10:48.001Z'
+    updated2 = '2015-09-23T08:00:40Z'
+    entry = '<entry><summary>{}</summary><updated>{}</updated></entry>'
```

Comment:
FWIW: `twisted.web.template` can produce valid XML if you find yourself needing to do a lot of this.